database/init.sql

```diff
@@ -10,4 +10,20 @@ CREATE TABLE Users (
     PRIMARY KEY (id)
 );
 
+CREATE TABLE Message (
+    id CHAR(36),
+    text VARCHAR(1024),
+    room_id INT,
+    timestamp INT,
+    user_id CHAR(36),
```

Comment:
```suggestion
    user_id CHAR(36),
```
Should this be a foreign key?